index.html

```diff
@@ -2326,6 +2327,101 @@ <h2>Definition of Roles</h2>
 				</tbody>
 			</table>
 		</div>
+		<div class="role" id="comment">
+			<rdef>comment</rdef>
+			<div class="role-description">
+				<p>A comment contains a remark expressing a reaction to other content within the page.</p>
+				<p>The relationship of comments to other content SHOULD be exposed:</p>
+				<ol>
+					<li>When the comment refers to other content within the page, the source content SHOULD use <pref>aria-details</pref> to point to the related comment.</li>
+					<li>When the comment is a reply to another comment:
```

Comment:
```suggestion
					<li>If the comment is a reply to another comment:
```

(As above - just a suggestion).